Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -3922,8 +3925,13 @@ def panorama_register_user_tag_command(args: dict):
         raise Exception('The panorama-register-user-tag command is only available for PAN-OS 9.X and above versions.')
     tag = args['tag']
     users = argToList(args['Users'])
+    # if not given, timeout will be 0 (never expires)
+    timeout = int(args.get('timeout', '0'))
```

Comment:
```suggestion
    timeout = arg_to_number(args.get('timeout', '0'))
```

Reply:
Changing both here and for the same thing in the `panorama_register_ip_tag_command` function.